CHANGELOG.md

```diff
@@ -17,6 +17,10 @@ changes.
 - Remove hydra-tools package. Move functionality to generate hydra keys to the
   hydra-node executable.
 
+- **BREAKING** Changes to `hydra-node` state persistency:
```

Comment:
The fact that the golden tests are only removing `previous` and still pass, is proof that this is in fact a backward compatible change! Well done!